### PR TITLE
Updates to scikit-learn wrapper in nltk.classify

### PR DESCRIPTION
**Pull request by larsmans**
- Dropped support for scikit-learn 0.11 in the example;
- added missing docstrings and generally improved the docs.

### CODE REVIEW

File: nltk/classify/scikitlearn.py [hunk 1]
```diff
@@ -69,24 +68,42 @@ def __repr__(self):
         return "<SklearnClassifier(%r)>" % self._clf
 
     def batch_classify(self, featuresets):
+        """Classify a batch of samples.
+
+        :param labeled_featuresets: A list of featuresets, each a dict
+            mapping strings to either numbers or booleans.
+        :return: The predicted class label for each input sample.
+        :rtype: List.
```
Comment (kmike) [inline nltk/classify/scikitlearn.py]: I think it is better to use type name as-is in `:rtype:` declarations ('list', not 'List')

**Comment by kmike** [comment]
Looks good, thanks! +1 for dropping support for scikit-learn 0.11.

File: nltk/classify/scikitlearn.py (hunk 1 above)
Comment (larsmans) [inline nltk/classify/scikitlearn.py]: Force-pushed a new version with this changed.

**Comment by stevenbird** [comment]
Thanks

**Pull request merged**